_projects/husky.md

```diff
@@ -0,0 +1,12 @@
+---
+title: husky
+description: Git hooks made easy
+repository: https://github.com/typicode/husky
+language: JavaScript
+developer:
+  name: Typicode
+  homepage: https://github.com/typicode
+version: 5.0.0
```

Comment:
Are you using Parity 5.0.0? The latest is [7.0.0](https://paritylicense.com/versions/7.0.0).

Reply:
My bad, I had husky v5.0 in mind when filling this field. Fixed :+1: :slightly_smiling_face:

Reply:
It's not you. It's confusing. Another PR did the same thing.

I will merge this, deploy, and then go back and fix the field name.